content/hi/docs/reference/glossary/app-container.md

```diff
@@ -0,0 +1,20 @@
+---
+title: ऐप कंटेनर (App Container)
+id: app-container
+date: 2019-02-12
+full_link:
+short_description: >
+  एक कंटेनर एक कार्यभार का हिस्सा चलाने के लिए प्रयोग किया जाता है। प्रारंभिक कंटेनर की तुलना में।
+
+aka:
+tags:
+- workload
+---
+एप्लिकेशन {{<glossary_tooltip text="पात्र" term_id="container" >}} (या ऐप कंटेनर) {{<glossary_tooltip text="पॉड" term_id="pod" >}} में, {{<glossary_tooltip text ="इनिट कंटेनर" term_id="init-container" >}} स्टार्टअप पूरा होने के बाद शुरू होता है।
+
+<!--more-->
+
+एक  इनिट कंटेनर के साथ, संपूर्ण
+आप {{< glossary_tooltip text="कार्यभार" term_id="workload" >}} के लिए महत्वपूर्ण आरंभीकरण विवरण अलग कर सकते हैं।
```

Comment:
Still not sure about कार्यभार and I presume you meant app workload 

```suggestion
ऐप {{< glossary_tooltip text="कार्यभार" term_id="workload" >}} के लिए महत्वपूर्ण आरंभीकरण विवरण अलग कर सकते हैं।
```

Reply:
If it was intended to mean you, in a respectful way, then the structure of this sentence needs to be modified as below.

एक इनिट कंटेनर के साथ, आप संपूर्ण  {{< glossary_tooltip text="कार्यभार" term_id="workload" >}} के लिए महत्वपूर्ण आरंभीकरण विवरण अलग कर सकते हैं।